components/brave_new_tab_ui/containers/newTab/footerInfo.tsx

```diff
@@ -21,6 +21,13 @@ interface Props {
 }
 
 export default class FooterInfo extends React.Component<Props, {}> {
+
+  onKeyPressSettings = (e: React.KeyboardEvent<HTMLButtonElement>) => {
```

Comment:
can we name it `event` to make it explicit? please change on other places as well